client/me/purchases/billing-history/receipt.jsx

```diff
@@ -310,7 +350,7 @@ function ReceiptLabels() {
 				id="billing-history__billing-details-description"
 			>
 				{ translate(
-					'Use this field to add your billing information (eg. VAT number, business address) before printing.'
+					'Use this field to add your billing information (eg. business address) before printing.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.